api/src/main/java/org/wildfly/extras/transformer/ArchiveTransformer.java

```diff
@@ -125,35 +122,18 @@ protected ResourceTransformer newResourceTransformer() throws IOException {
         throw new UnsupportedOperationException();
     }
 
-    private static void safeClose(final Closeable c) {
-        try {
-            if (c != null) c.close();
-        } catch (final Throwable t) {
-            // ignored
+    private static void readBytes(final InputStream is, final byte[] clazz) throws IOException {
+        int offset = 0;
+        while (offset < clazz.length) {
+            offset += is.read(clazz, offset, clazz.length - offset);
         }
     }
 
-    private static void readBytes(final InputStream is, final byte[] clazz, final boolean closeStream) throws IOException {
-        try {
-            int offset = 0;
-            while (offset < clazz.length) {
-                offset += is.read(clazz, offset, clazz.length - offset);
-            }
-        } finally {
-            if (closeStream) {
-                safeClose(is);
-            }
-        }
+    private static void writeBytes(final OutputStream os, final byte[] clazz) throws IOException {
+        os.write(clazz);
     }
 
-    private static void writeBytes(final OutputStream os, final byte[] clazz, final boolean closeStream) throws IOException {
-        try {
-            os.write(clazz);
-        } finally {
-            if (closeStream) {
-                safeClose(os);
-            }
-        }
+    public boolean supportClassTransformation() {
```

Comment:
Is this supposed to return false file transformers that do not support transforming standalone class files?  Should it install be called `supportClassFileTransformation` or `canTransformIndividualClassFile`?

Reply:
yes you are correct :) I'll rename it